build/base.sh

```diff
@@ -58,7 +58,11 @@ BASE_OBJDIR="$(make -C${SRCDIR}/release -V .OBJDIR)"
 setup_stage "${BASE_OBJDIR}/${BASE_DISTDIR}"
 
 # remove older object archives, too
-BASE_OBJ=$(make -C${SRCDIR}/release -V .OBJDIR)/base.txz
+if [ ${PRODUCT_HOST} != ${PRODUCT_ARCH} ]; then
```

Comment:
hmm, ok, not very portable but ok... I'm simply wondering if this check is actually correct and if this breaks compat for 11.2 cross-building which was supposed to be working fine.

Reply:
Yes, this will definitely break cross compiling on BSD 11.2 (didn't thought about this).

So whats the solution for this?
Include an exception for 11.2, or wait until 11.2 (OPNsense 19.7) is obsolete?

Reply:
Well .... screwed up my git again, let me fix that